VirtualSatelliteCAD/commands/command_import.py

```diff
@@ -28,19 +28,35 @@
 import FreeCADGui
 from module.environment import Environment, ICON_IMPORT
 from commands.command_definitions import COMMAND_ID_IMPORT_2_FREECAD
+from PySide2.QtWidgets import QFileDialog
+from json_io.json_importer import JsonImporter
+import os
 
 
-class CommandExport:
+class CommandImport:
     def Activated(self):
-        FreeCAD.Console.PrintMessage("Calling the json_io\n")
+        FreeCAD.Console.PrintMessage("Calling the importer\n")
+
+        # call pyqt dialog: returns (filename, filter)
+        filename = QFileDialog.getOpenFileName(
+            None,  # ui parent
+            "Open JSON file",  # dialog caption
+            Environment.get_appdata_module_path(),
+            "JSON(*.json)")[0]  # filter
+
+        if filename != '':
+            FreeCAD.Console.PrintMessage(f"Selected file '{filename}'\n")
+
+            json_importer = JsonImporter(Environment.get_appdata_module_path() + os.sep)

```

Comment:
Why do we have to hand over this appdata_module_path?

Reply:
Created an issue for this: https://github.com/virtualsatellite/VirtualSatellite4-FreeCAD-mod/issues/19